commands/scanandfixrepos.go

```diff
@@ -22,43 +18,22 @@ type ScanAndFixRepositories struct {
 func (saf *ScanAndFixRepositories) Run(configAggregator utils.FrogbotConfigAggregator, client vcsclient.VcsClient) error {
 	var errList strings.Builder
 	for repoNum := range configAggregator {
-		if err := saf.scanAndFixSingleRepository(&configAggregator[repoNum], client); err != nil {
-			errList.WriteString(fmt.Sprintf("repository %s returned the following error: \n%s\n", configAggregator[repoNum].RepoName, err.Error()))
-		}
+		saf.scanAndFixSingleRepository(&configAggregator[repoNum], client, errList)
 	}
 	if errList.String() != "" {
 		return errors.New(errList.String())
 	}
 	return nil
 }
 
-func (saf *ScanAndFixRepositories) scanAndFixSingleRepository(repoConfig *utils.FrogbotRepoConfig, client vcsclient.VcsClient) error {
+func (saf *ScanAndFixRepositories) scanAndFixSingleRepository(repoConfig *utils.FrogbotRepoConfig, client vcsclient.VcsClient, errList strings.Builder) {
```

Comment:
It's better to have this function return an error, instead of having `errList` received as an outgoing argument.